ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileClustersPage.tsx

```diff
@@ -66,8 +67,9 @@ function ProfileClustersPage() {
     const { searchFilter, setSearchFilter } = useURLSearch();
 
     const fetchProfileClusters = useCallback(() => {
+        const regexSearchFilter = addRegexPrefixToFilters(searchFilter, [CLUSTER_QUERY]);
```

Comment:
This should also just be `CHECK_NAME_QUERY`. Since cluster filter is based off autocomplete, I don't think we want to substring match it.

Reply:
Nice catch. I should add the `CHECK_NAME_QUERY`, but why shouldn't I leave the `CLUSTER_QUERY`? It is based on autocomplete, but the user can put some partial string and search it using the arrow button

This is what happens if we take it out

<img width="1437" alt="Screenshot 2024-06-26 at 3 45 39 PM" src="https://github.com/stackrox/stackrox/assets/4805485/dfcc51f8-1b2d-4343-b40d-d2df37035a83">

Reply:
Ah, I forgot you could put in your own value. I'm fine with keeping `CLUSTER_QUERY` in that case. My only hesitancy is if I specifically click `production` in the autocomplete list, I'm not sure I'd expect `production-blah` to also show up as well.

Reply:
Although I guess that would match regardless of the regex, wouldn't it?

Reply:
Ok lets keep it the way you imagined and discuss this with UX after the release 👍🏼 I think this is something we need to discuss more.